cmd/podman/cliconfig/config.go

```diff
@@ -80,6 +80,15 @@ type CommitValues struct {
 	Quiet   bool
 }
 
+type PatchValues struct {
+	PodmanCommand
+	DST     string
```

Comment:
I think we'd prefer to capitalize only the first character for this and PATCH - `Dst` and `Patch`